tests/utils_tests/test_feedgenerator.py

```diff
@@ -121,3 +121,12 @@ def test_feed_with_feed_url_gets_rendered_with_atom_link(self):
         self.assertIn('<atom:link', feed_content)
         self.assertIn('href="/feed/"', feed_content)
         self.assertIn('rel="self"', feed_content)
+
+    def test_latest_post_date_returns_utc_time(self):
+        """
+        Test to make sure latest_post_date returns time in UTC
```

Comment:
Just a nitpick: The test name looks already readable to me so I'd drop the docstring.